code/game/objects/structures/morgue.dm

```diff
@@ -101,21 +89,21 @@
 
 /obj/structure/morgue/attackby(obj/P, mob/user)
 	if(P.ispen())
-		var/t = input(user, "What would you like the label to be?", text("[]", src.name), null)  as text
+		var/t = input(user, "What would you like the label to be?", "[src.name]") as text
 		if(user.get_active_hand() != P)
 			return
-		if((!in_range(src, usr) && src.loc != user))
+		if((!in_range(src, usr) > 1 && src.loc != user))
 			return
```

Comment:
i dunno if this works with the ! in front, looks messy

Reply:
you resolved this one without actually changing it

Reply:
yes, it works